src/Illuminate/Support/Traits/Macroable.php

```diff
@@ -80,11 +80,13 @@ public static function __callStatic($method, $parameters)
             ));
         }
 
-        if (static::$macros[$method] instanceof Closure) {
-            return call_user_func_array(Closure::bind(static::$macros[$method], null, static::class), $parameters);
+        $macro = static::$macros[$method];
+
+        if ($macro instanceof Closure) {
+            return call_user_func_array(Closure::bind($macro, null, static::class), $parameters);
         }
 
-        return call_user_func_array(static::$macros[$method], $parameters);
+        return call_user_func_array($macro, $parameters);
```

Comment:
Can we replace `call_user_func_array` with just `$macro(...$parameters)`?

Reply:
Done. Good suggestion 👍